src/test/java/com/algorand/algosdk/transaction/TestTransaction.java

```diff
@@ -53,4 +55,19 @@ public void testSerializationMsgpack() throws Exception {
         Assert.assertEquals(tx.genesisHash, o.genesisHash);
     }
 
+    @Test
+    public void testSerializationAssetConfig() throws Exception {
+        Address addr = new Address("BH55E5RMBD4GYWXGX5W5PJ5JAHPGM5OXKDQH5DC4O2MGI7NW4H6VOE4CP4");
+        byte[] gh = Encoder.decodeFromBase64("SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=");
+        Transaction tx = new Transaction(addr, BigInteger.valueOf(10), BigInteger.valueOf(322575), BigInteger.valueOf(323575), null, "", new Digest(gh), addr, BigInteger.valueOf(1234), addr, addr, addr, addr);
```

Comment:
How do I know that asset transaction's constructor is called? please name some params to make it explicit.